packages/typespec-test/test/batch_modular/generated/typespec-ts/review/batch.api.md

```diff
@@ -161,22 +161,22 @@ export class BatchClient {
     getTaskFileProperties(jobId: string, taskId: string, filePath: string, options?: GetTaskFilePropertiesOptions): Promise<void>;
     jobScheduleExists(jobScheduleId: string, options?: JobScheduleExistsOptions): Promise<void>;
-    listApplications(options?: ListApplicationsOptions): Promise<ApplicationListResult>;
-    listCertificates(options?: ListCertificatesOptions): Promise<CertificateListResult>;
-    listJobPreparationAndReleaseTaskStatus(jobId: string, options?: ListJobPreparationAndReleaseTaskStatusOptions): Promise<BatchJobListPreparationAndReleaseTaskStatusResult>;
-    listJobs(options?: ListJobsOptions): Promise<BatchJobListResult>;
-    listJobSchedules(options?: ListJobSchedulesOptions): Promise<BatchJobScheduleListResult>;
-    listJobsFromSchedule(jobScheduleId: string, options?: ListJobsFromScheduleOptions): Promise<BatchJobListResult>;
-    listNodeExtensions(poolId: string, nodeId: string, options?: ListNodeExtensionsOptions): Promise<NodeVMExtensionList>;
-    listNodeFiles(poolId: string, nodeId: string, options?: ListNodeFilesOptions): Promise<NodeFileListResult>;
-    listNodes(poolId: string, options?: ListNodesOptions): Promise<BatchNodeListResult>;
-    listPoolNodeCounts(options?: ListPoolNodeCountsOptions): Promise<PoolNodeCountsListResult>;
-    listPools(options?: ListPoolsOptions): Promise<BatchPoolListResult>;
-    listPoolUsageMetrics(options?: ListPoolUsageMetricsOptions): Promise<PoolListUsageMetricsResult>;
+    listApplications(options?: ListApplicationsOptions): PagedAsyncIterableIterator<BatchApplication>;
+    listCertificates(options?: ListCertificatesOptions): PagedAsyncIterableIterator<BatchCertificate>;
+    listJobPreparationAndReleaseTaskStatus(jobId: string, options?: ListJobPreparationAndReleaseTaskStatusOptions): PagedAsyncIterableIterator<JobPreparationAndReleaseTaskExecutionInformation>;
+    listJobs(options?: ListJobsOptions): PagedAsyncIterableIterator<BatchJob>;
+    listJobSchedules(options?: ListJobSchedulesOptions): PagedAsyncIterableIterator<BatchJobSchedule>;
+    listJobsFromSchedule(jobScheduleId: string, options?: ListJobsFromScheduleOptions): PagedAsyncIterableIterator<BatchJob>;
+    listNodeExtensions(poolId: string, nodeId: string, options?: ListNodeExtensionsOptions): PagedAsyncIterableIterator<NodeVMExtension>;
+    listNodeFiles(poolId: string, nodeId: string, options?: ListNodeFilesOptions): PagedAsyncIterableIterator<NodeFile>;
+    listNodes(poolId: string, options?: ListNodesOptions): PagedAsyncIterableIterator<BatchNode>;
+    listPoolNodeCounts(options?: ListPoolNodeCountsOptions): PagedAsyncIterableIterator<PoolNodeCounts>;
+    listPools(options?: ListPoolsOptions): PagedAsyncIterableIterator<BatchPool>;
+    listPoolUsageMetrics(options?: ListPoolUsageMetricsOptions): PagedAsyncIterableIterator<PoolUsageMetrics>;
     listSubTasks(jobId: string, taskId: string, options?: ListSubTasksOptions): Promise<BatchTaskListSubtasksResult>;
```

Comment:
Just curious why this is not updated?  Is it a paging operation?

Reply:
I checked this will not be devired as paging operation and I will confirm with service team any reason not do that.